### PR TITLE
Update tutorials step  1 & 6, Closes #284

### LINKED ISSUE #284: Use tee to simultanesously print and send the transaction object
Use `tee` to simultanesously print and send the transaction object:

`node print_sendable_cashback.js | tee >(curl -X POST -H "Content-Type: application/json" -d @- localhost:4000/api/transactions) # displays a raw transaction on the console`

Then search for transaction by type, not id (id changes due to dynamic timestamp).

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #284

### CODE REVIEW

File: tutorials/cashback.md [hunk 1]
```diff
@@ -17,22 +18,46 @@ First, let's create the root folder for the Cashback App and initialize the proj
...
+| Node.js          | 10.15.3 |
+| PostgreSQL       | 10+     |
+| Redis (optional) | 5+      |
+| Python           | 2       |
+
+
+> If you miss some of the dependencies, please go to [Lisk SDK - Pre-Install](../../lisk-sdk/introduction.md#pre-installation) and follow the pre-installation steps for the SDK.
 
 ```bash
+npm init # initialize the manifest file of the project
```
Comment (RachBLondon) [inline tutorials/cashback.md:45]:
```suggestion
npm init --yes # initialize the manifest file of the project
```

File: tutorials/hello-world.md [hunk 2]
```diff
@@ -21,22 +21,49 @@ First, let's create the root folder for the Hello World App and initialize the p
 ```bash
 mkdir hello_world # create the root folder for the blockchain application
 cd hello_world # navigate into the root folder
-npm init # initialize the manifest file of the project
```
Comment (RachBLondon) [inline tutorials/hello-world.md:24]: Shouldn't we leave this in?

```suggestion
npm init --yes # initialize the manifest file of the project
```

File: tutorials/cashback.md (hunk 1 above)
Comment (Tschakki) [reply tutorials/cashback.md:45]: This has been already merged into `feature/marketing` branch, doesn't need to be changed here, too.

Comment (RachBLondon) [reply tutorials/cashback.md:45]: 👍

**Review by RachBLondon — APPROVED** [review]

**Pull request merged**